gallery/src/components/NxCombobox/NxComboboxErrorExample.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useCallback, useState } from 'react';
+import { useDebounceCallback } from '@react-hook/debounce';
+import { NxCombobox, DataItem, NX_SEARCH_DROPDOWN_DEBOUNCE_TIME, NxFormGroup }
+  from '@sonatype/react-shared-components';
+
+export default function NxComboboxErrorExample() {
+  const [loading, setLoading] = useState(false),
+      [query, setQuery] = useState(''),
+      [error, setError] = useState<string | null>(null);
+
+  function onSelect({ displayName }: DataItem<number>) {
+    alert('Selected ' + displayName);
+  }
+
+  const executeQuery = useDebounceCallback(useCallback(function executeQuery(query: string) {
+    setError(`Error executing query ${query}`);
+    setLoading(false);
+  }, [query]), NX_SEARCH_DROPDOWN_DEBOUNCE_TIME);
+
+  function onSearchTextChange(query: string) {
+    setQuery(query);
+  }
+
+  function onSearch(query: string) {
+    setError(null);
+
+    if (query !== '') {
+      setLoading(true);
+      executeQuery(query);
+    }
+  }
+
+  return (
+    <NxFormGroup label="Combobox">
```

Comment:
Do we need the `NxFormGroup` in the example? The `NxTextInput` examples just have the bare component

Reply:
The label is presented in mockups, hence, I would like to wrap the combobox within the `NxFormGroup` which also provided a label for the combobox.

Reply:
> The label is presented in mockups, hence, I would like to wrap the combobox within the NxFormGroup which also provided a label for the combobox.

Our typical pattern is to have the examples on the components own page be just the component, and then to verify that it lays out correctly with a label and relative to other form fields, we should also add it to the form on the Form Layout Examples page.